packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -899,7 +911,7 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     const person = this.personDetails as User;
     const user = this._me.userPrincipalName;
     const chatInput = this._chatInput;
-    if (person.userPrincipalName === user) {
+    if (person && person.userPrincipalName === user) {
```

Comment:
This can be simplified using optional chaining
```suggestion
    if (person?.userPrincipalName === user) {
```

Reply:
I've seen a couple of comments like this lately. Could we add linting to warn us?

Reply:
maybe, I want to get us off tslint and on to eslint as a first point on linting, then we can look at adding new rules